charms/kfp-profile-controller/requirements.in

```diff
@@ -1,3 +1,3 @@
+charmed-kubeflow-chisme @ git+https://github.com/canonical/charmed-kubeflow-chisme.git@add-base-charm
```

Comment:
You can use chisme's release instead of this branch and remove git from the charmcraft build-packages https://github.com/canonical/kfp-operators/blob/main/charms/kfp-viewer/requirements.in#L3